packages/wallets/src/features/cashier/modules/WithdrawalVerification/WithdrawalVerificationRequest/__tests__/WithdrawalVerificationRequest.spec.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import WithdrawalVerificationRequest from '../WithdrawalVerificationRequest';
+
+describe('WithdrawalVerificationRequest', () => {
+    test('should render component correctly', () => {
+        render(<WithdrawalVerificationRequest sendEmail={jest.fn()} />);
+
+        expect(screen.getByText('Please help us verify your withdrawal request.')).toBeInTheDocument();
+
+        const sendEmailButton = screen.getByText('Send email');
+        expect(sendEmailButton).toBeInTheDocument();
```

Comment:
hey @lubega-deriv 
could we use getByRole for button elements?
pls refer [this](https://testing-library.com/docs/queries/about#:~:text=Yes-,Priority%E2%80%8B,button%27%2C%20%7Bname%3A%20/submit/i%7D).%20Check%20the%20list%20of%20roles.,-getByLabelText%3A%20This%20method)

Reply:
Updated